dpctl-capi/source/dpctl_sycl_queue_interface.cpp

```diff
@@ -459,17 +462,21 @@ void DPCTLQueue_Memcpy(__dpctl_keep const DPCTLSyclQueueRef QRef,
                        size_t Count)
 {
     auto Q = unwrap(QRef);
-    auto event = Q->memcpy(Dest, Src, Count);
-    event.wait();
+    if (Q) {
```

Comment:
👍 

Should an error be recorded if `Q` is null?

Reply:
Maybe, but we return `void`, so I figured I'd just prevent the crash. 

This can be addressed due to the pass of error handling improvements.